src/components/v5/common/WidgetCards/partials/WidgetContent.tsx

```diff
@@ -2,27 +2,32 @@ import { type Icon } from '@phosphor-icons/react';
 import React, { type FC, type PropsWithChildren } from 'react';
 
 import { WidgetIconWrapper } from './WidgetIconWrapper.tsx';
+import { WidgetSubTitle } from './WidgetSubTitle.tsx';
 import { WidgetTitle } from './WidgetTitle.tsx';
 
 interface WidgetContentProps {
   Icon?: Icon;
   title?: React.ReactNode;
+  subTitle?: React.ReactNode;
 }
 
 export const WidgetContent: FC<PropsWithChildren<WidgetContentProps>> = ({
   Icon,
   title,
+  subTitle,
   children,
 }) => (
   <>
     {Icon ? (
       <WidgetIconWrapper Icon={Icon}>
         {title && <WidgetTitle title={title} />}
+        {subTitle && <WidgetSubTitle title={subTitle} />}
         {children}
```

Comment:
Rather than separate `WidgetTitle` and `WidgetSubTitle` components, maybe we could have a `WidgetInner` component or something? As lines 23-25 and 29-31 are identical.

Or `WidgetIconWrapper` could be moved up a component to wrap `WidgetContent` conditionally there?